x/cronos/events/events.go

```diff
@@ -22,23 +25,45 @@ var (
 		banktypes.AttributeKeyMinter:     ConvertAccAddressFromBech32,
 		banktypes.AttributeKeyBurner:     ConvertAccAddressFromBech32,
 	}
+	IcaValueDecoders = ValueDecoders{
+		channeltypes.AttributeKeyChannelID: ReturnStringAsIs,
+		channeltypes.AttributeKeyPortID:    ReturnStringAsIs,
+		cronoseventstypes.AttributeKeySeq:  ConvertUint64,
+	}
 )
 
 func init() {
-	var ibcABI abi.ABI
-	if err := ibcABI.UnmarshalJSON([]byte(generated.RelayerModuleMetaData.ABI)); err != nil {
+	var relayerABI abi.ABI
+	if err := relayerABI.UnmarshalJSON([]byte(relayer.RelayerModuleMetaData.ABI)); err != nil {
+		panic(err)
+	}
+	RelayerEvents = NewEventDescriptors(relayerABI)
+
+	var icaABI abi.ABI
+	if err := icaABI.UnmarshalJSON([]byte(ica.ICAModuleMetaData.ABI)); err != nil {
 		panic(err)
 	}
-	IBCEvents = NewEventDescriptors(ibcABI)
+	IcaEvents = NewEventDescriptors(icaABI)
+}
+
+func RelayerConvertEvent(event sdk.Event) (*ethtypes.Log, error) {
+	if event.Type == sdk.EventTypeMessage {
+		return nil, nil
+	}
+	desc, ok := RelayerEvents[event.Type]
+	if !ok {
+		return nil, nil
+	}
+	return desc.ConvertEvent(event.Attributes, RelayerValueDecoders.WithDefaultDecoder(ReturnStringAsIs))
```

Comment:
I don't like the idea of default decoder, it'll bloat the logs

Reply:
Yea, since need relayer change I tend to clean it after this